src/server/verification/api/ZoomAPI.js

```diff
@@ -313,6 +310,30 @@ class ZoomAPI {
     return databaseIndex
   }
 
+  async _enrollmentRequest(operation, enrollmentIdentifier, customLogger = null) {
+    let response
+
+    try {
+      response = await this.http[operation]('/enrollment-3d/:enrollmentIdentifier', {
+        customLogger,
+        params: { enrollmentIdentifier }
+      })
+    } catch (exception) {
+      const { message } = exception
+
+      if (/(no\s+entry\s+found|no\s+records\s+were)/i.test(message)) {
```

Comment:
no error code or some better way to recognize a specific error than a regex?

Reply:
Unfortunately no. v8 API had code / subCode response fields, in v9 they were removed. 
In a theory we could add them by modifying Java sources of the custom server https://github.com/GoodDollar/FaceTecSDK-custom-server/tree/master/FaceTecSDK-custom-server-9.0.8/src/main/java/com/facetec/customserver

But it's not critical for now - as we have access to the sources and could keep the same error messages. 
You could add an enhancement task for that if you want. For now the main priority is update to v9 + make FV working on native